spec.html

```diff
@@ -4197,7 +4197,7 @@ <h1>ToBoolean ( _argument_ )</h1>
               String
             </td>
             <td>
-              If _argument_ is the empty String (its length is zero), return *false*; otherwise return *true*.
```

Comment:
No, the length of a String is a mathematical value.

Reply:
ah right, good call.